tutorials/3-run-node/index.md

```diff
@@ -41,52 +41,135 @@ $ cd cosmos-sdk
 Make sure you are using the same version used at the time of writing:
 
 ```sh
-$ git checkout v0.42.6
+$ git checkout v0.45.4
 ```
 
-Then build `cosmos-sdk`:
+Then build `cosmos-sdk`. If you use Docker, with the help of a ready-made [multi-stage creation](https://github.com/cosmos/cosmos-sdk/blob/v0.45.4/Dockerfile#L37), you create a new Docker image that contains the compiled `simd`:
+
+<CodeGroup>
+
+<CodeGroupItem title="Local">
 
 ```sh
 $ make build
 ```
 
-The build takes a few minutes and creates a `build` folder and a `simapp` binary named `simd`:
+The build takes a few minutes and creates a `build` folder and a `simapp` binary named `simd`.
+
+</CodeGroupItem>
+
+<CodeGroupItem title="Docker">
 
 ```sh
-$ ls build
+$ docker build . -t simd:v0.45.4
+```
+
+</CodeGroupItem>
+
+</CodeGroup>
+
+Confirm that you got what you expected:
+
+<CodeGroup>
+
+<CodeGroupItem title="Local">
+
+```sh
+$ ./build/simd version
```

Comment:
It is a better way to check than `ls build` but would it not be better to check before `docker build . -t simd:v0.45.4` and after `make build`?

Reply:
You want to check that the `build` folder is empty before we build?

Reply:
No sorry, `./build/simd version` comes right after `make build`. `docker build . -t simd:v0.45.4` is not between, because you present two tracks.